app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayController.kt

```diff
@@ -276,13 +276,16 @@ class DefaultTabsTrayController(
      */
     @OptIn(DelicateAction::class)
     override fun forceTabsAsInactive(tabs: Collection<TabSessionState>, numOfDays: Long) {
-        tabs.forEach { tab ->
-            val daysSince = System.currentTimeMillis() - TimeUnit.DAYS.toMillis(numOfDays)
-            browserStore.apply {
-                dispatch(LastAccessAction.UpdateLastAccessAction(tab.id, daysSince))
-                dispatch(DebugAction.UpdateCreatedAtAction(tab.id, daysSince))
+        val currentTabId = browserStore.state.selectedTabId
+        tabs
+            .filterNot { it.id == currentTabId }
```

Comment:
❓ Question: Should we add tests for it?

💡 Suggestion: Maybe we can add checks that it should only work on debugging mode and does nothing in other modes, not for this PR.

Reply:
I was thinking of adding tests but considering it is a feature used only for debugging, I didn't really see them necessary. Not sure if the suggestion is for the tests or for the feature. The check for the force inactive feature is actually implemented here: https://searchfox.org/mozilla-mobile/source/fenix/app/src/main/java/org/mozilla/fenix/tabstray/browser/SelectionMenu.kt#48

Reply:
Fair point, although seeing that it did break, and you fixed it here, even for debugging adding the tests would avoid this from happening in the future, i leave it to your better judgement :)
Suggestion was for the feature, the check is somewhere else and from reading the comments it seemed like action can be used by mistake, adding a check there would prevent that. Again, not essential for this PR, but might prevent this super user feature from leaking into the release ;)

Reply:
Kudos to @Mugurell for helping, three new tests have been added which would ensure future stability for this feature.